uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification-badge.component.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/components'
+], function (components) {
+  components.component('leaveNotificationBadge', {
+    bindings: {
+      eventName: '<'
```

Comment:
i was thinking that this name should be more descriptive ("event name" of what? event for what?) something like
`countTrigger`?